<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="navigation_buttonAvailability">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>380</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Screenshot 2023-05-06 170619.png</normaloff>Screenshot 2023-05-06 170619.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="navigation_buttonGenerateTimetable">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>500</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Screenshot 2023-05-06 170612.png</normaloff>Screenshot 2023-05-06 170612.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="navigation_buttonGroupsAndRooms">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>440</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Screenshot 2023-05-06 170631.png</normaloff>Screenshot 2023-05-06 170631.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="interface_label5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>681</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>## Groups</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>-30</y>
      <width>20</width>
      <height>891</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>-30</y>
      <width>20</width>
      <height>971</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="interface_label6">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>671</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>## Cabinets</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="groups_buttonAddGroup">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>120</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ADD GROUP</string>
    </property>
   </widget>
   <widget class="QTableView" name="groups_tableGroups">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>611</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="cabinets_tableCabinets">
    <property name="geometry">
     <rect>
      <x>718</x>
      <y>170</y>
      <width>611</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cabinets_buttonAddCabinet">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>120</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ADD CABINET</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
